### PR TITLE
Ecs update

### PR DESCRIPTION
**Pull request by gabcoyne**

### CODE REVIEW

File: aws/prefect/prefect-agent-on-ecs/network-config.yaml.tftpl [hunk 1]
```diff
@@ -1,11 +0,0 @@
-networkConfiguration:
```
Comment (jamiedick) [inline aws/prefect/prefect-agent-on-ecs/network-config.yaml.tftpl:1]: why was this removed?

Comment (jamiedick) [reply aws/prefect/prefect-agent-on-ecs/network-config.yaml.tftpl:1]: nvm i see it down below

File: aws/prefect/prefect-agent-on-ecs/s3.tf [hunk 2]
```diff
@@ -19,18 +19,30 @@ resource "aws_s3_bucket_public_access_block" "bucket_block" {
...
+  etag = filemd5("${path.module}/task-definition.yaml.tftpl")
+}
+
+resource "aws_s3_object" "network_config" {
+  depends_on = [
+    aws_s3_bucket.prefect_ecs_config
+  ]
+  bucket  = aws_s3_bucket.prefect_ecs_config.id
+  key     = "network-config.yaml"
+  content = yamlencode({ "networkConfiguration" : { "awsvpcConfiguration" : { "subnets" : [for subnet in var.subnet_ids : subnet], "securityGroups" : [aws_security_group.sg.id], "assignPublicIp" : "ENABLED" } } })
```
Comment (jamiedick) [inline aws/prefect/prefect-agent-on-ecs/s3.tf:46]: just want to verify you tested this?  this is very particular and just want to make sure its been tested

**Review by jamiedick — COMMENTED** [review]
looks good to me. just 1 questioN!

File: aws/prefect/prefect-agent-on-ecs/s3.tf (hunk 2 above)
Comment (gabcoyne) [reply aws/prefect/prefect-agent-on-ecs/s3.tf:46]: Yep, I am going to parameterize the public IP thing though. The client network I was working on was not in line with aws well architected design.

**Review by zzstoatzz — APPROVED** [review]
